week-1/articles/installfest.md

```diff
@@ -5,25 +5,37 @@ Welcome to your  software program!
 We will use several tools during class, please see the instructions for installing the specific tools you will need based on your operating system.
 
 Please note that we can support students using the following operating systems:
-  - Windows 10+
-  - Apple MacOS Mojave 10.14+
+
+- Windows 10+
+- Apple MacOS Mojave 10.14+
+
+## Online User Accounts (Applicable to all Operating Systems)
+
+### Github User Account
+
+1. Visit the Github Signup page at <https://github.com/join>
+2. Create a user account and select a unique user name.
+3. **Optional**: Read about [Getting started with Github](https://docs.github.com/en/github/getting-started-with-github)
+
+#### Heroku User Account
+
+For the Heroku Command Line Tools to work correctly you will need a Heroku user account.
+
+1. Visit https://signup.heroku.com/
+2. Create a new user account
+3. **Optional**: Read about [Getting started with Node.js on Heroku](https://devcenter.heroku.com/articles/getting-started-with-nodejs)
 
 ## Windows Installation
 
 The instructions below will walk you through installing the following tools:
```

Comment:
I would also mention that these tools are for a Win install only. Mac users are further along the document. Matt told me that this was an issue in the past.